src/import_export/qif_import_gui.cpp

```diff
@@ -357,8 +360,18 @@ bool mmQIFImportDialog::mmReadQIFFile()
     wxLongLong start = wxGetUTCTimeMillis();
     wxLongLong interval = wxGetUTCTimeMillis() - start;
 
-    wxString accName = ""; //TODO: check Account check box
```

Comment:
:+1:

Reply:
Bad style to leave todos in the PR but I want to commit as is.